satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -262,6 +262,7 @@ model project (
 
     field name           text
     field description    text      ( updatable )
+    field usage_limit    int64     ( updatable )
     field partner_id     blob      ( nullable  )
     field owner_id       blob
```

Comment:
You can add specific queries only for `usage_limit` rather than querying for the whole project.

Reply:
I don't understand, do you mean that `usage_limit` should not be part of `project` model?

Reply:
You can use:

```
read one (
    select project.usage_limit
    where  project.id = ?
)
```

To avoid reading the whole row when querying data.